code/game/objects/items/storage/backpack.dm

```diff
@@ -606,15 +526,154 @@
+	new /obj/item/grenade/chem_grenade/teargas/moustache(src)
+	new /obj/item/disk/nuclear/fake(src)
+	new /obj/item/slimepotion/slime/sentience/nuclear(src)
+	new /obj/item/storage/box/syndie_kit/chameleon/broken(src)
+	new /obj/item/toy/cards/deck/syndicate(src)
+
+/obj/item/storage/backpack/duffelbag/syndie/bioterror
+	name = "bioterrorist duffel bag"
+	desc = "A large duffel bag for holding biohazardous material. Handle with extreme care"
+
+/obj/item/storage/backpack/duffelbag/syndie/bioterror/PopulateContents()
+	new /obj/item/reagent_containers/spray/chemsprayer/bioterror(src)
+	new /obj/item/storage/box/syndie_kit/bioterror(src)
+	new /obj/item/storage/box/syndie_kit/tuberculosisgrenade(src)
+
+/obj/item/storage/backpack/duffelbag/syndie/knight
+	name = "energy knight duffel bag"
+	desc = "A large duffel bag for holding electronic swordsman gear"
+
+/obj/item/storage/backpack/duffelbag/syndie/knight/PopulateContents()
+	new /obj/item/melee/transforming/energy/sword/saber(src)
+	new /obj/item/shield/energy(src)
+	new /obj/item/clothing/shoes/chameleon/noslip(src)
+	new /obj/item/autosurgeon/anti_stun(src)
+	new /obj/item/storage/box/syndie_kit/imp_adrenal(src)
+	new /obj/item/autosurgeon/thermal_eyes(src)
+
+/obj/item/storage/backpack/duffelbag/syndie/cqc
+	name = "cqc duffel bag"
+	desc = "A large duffel bag for holding the secrets of hand to hand combat"
+
+/obj/item/storage/backpack/duffelbag/syndie/cqc/PopulateContents()
+	new /obj/item/book/granter/martial/cqc(src)
+	new /obj/item/clothing/gloves/rapid(src)
+	new /obj/item/soap/syndie(src)
+	new /obj/item/clothing/glasses/phantomthief/syndicate(src)
+	new /obj/item/card/emag(src)
+	new /obj/item/storage/box/syndie_kit/throwing_weapons(src)
+
+/obj/item/storage/backpack/duffelbag/syndie/infiltrator/PopulateContents()
+	new /obj/item/gun/ballistic/automatic/pistol(src)
+	new /obj/item/suppressor(src)
+	new /obj/item/suppressor(src)
+	new /obj/item/ammo_box/magazine/m10mm(src)
+	new /obj/item/ammo_box/magazine/m10mm(src)
+	new /obj/item/ammo_box/magazine/m10mm(src)
+	new /obj/item/ammo_box/magazine/m10mm(src)
+	new /obj/item/switchblade(src)
+	new /obj/item/grenade/smokebomb(src)
+	new /obj/item/grenade/smokebomb(src)
+	new /obj/item/card/emag(src)
+	new /obj/item/camera_bug(src)
+	new /obj/item/multitool/ai_detect(src)
+	new /obj/item/implanter/stealth(src)
+
+/obj/item/storage/backpack/duffelbag/syndie/engineer
+	name = "combat engineer duffel bag"
+	desc = "A large duffel bag for holding a lot of engineering supplies"
+
+/obj/item/storage/backpack/duffelbag/syndie/engineer/PopulateContents()
+	new /obj/item/storage/belt/military/engineer(src)
+	new /obj/item/clothing/shoes/magboots/syndie(src)
+	new /obj/item/encryptionkey/binary(src)
+	new /obj/item/gun/ballistic/automatic/shotgun/bulldog(src)
+	new /obj/item/ammo_box/magazine/m12g/slug(src)
+	new /obj/item/ammo_box/magazine/m12g/dragon(src)
+	new /obj/item/grenade/plastic/c4(src)
+	new /obj/item/grenade/plastic/x4(src)
```

Comment:
for an engineer this op bundle sure does lack any... engineering gear.

Reply:
I feel stupid for not giving it an RCD. I will change that